core/modules/filter/js/filter.js

```diff
@@ -209,8 +208,9 @@ Backdrop.behaviors.editorImageDialog = {
 
           // Display the library view.
           $('.editor-image-fields').removeClass('editor-image-fields-full');
-          $('.editor-image-library').css({ 'display': 'block' });
-          $('.form-item-image-directory').css({ 'display': 'block' });
+          $('.editor-image-library').load(Backdrop.settings.basePath + 'filter_image_library_ajax');
+          $('.editor-image-library').addClass('shown');
+          $('.form-item-image-directory').addClass('shown');
```

Comment:
Not sure about this `shown` class instead of toggling the `display` property. There's some chance of class name conflicts.